src/plots/cartesian/axes.js

```diff
@@ -2576,6 +2595,20 @@ axes.drawOne = function(gd, ax, opts) {
             }
         }
 
+        if(ax._shiftPusher) {
+            if(s === 'l') {
+                ax._fullDepth = Math.max(llbbox.height > 0 ? pos - llbbox.left : 0, outsideTickLen);
+            } else {
+                ax._fullDepth = Math.max(llbbox.height > 0 ? llbbox.right - pos : 0, outsideTickLen);
+            }
+            // TODO: Multiplying the approx depth seems to be a workaround for getting the default standoff?
+            if(ax.title.text !== fullLayout._dfltTitle[axLetter]) {
+                ax._fullDepth += (approxTitleDepth(ax) * 2) + (ax.title.standoff || 0);
```

Comment:
@alexcjohnson @archmoj wondering if you have any thoughts on this? I'm trying to account for the axis title here plus its standoff (user-defined or otherwise). I've had a hard time nailing down the variable that holds the default computed standoff. I've tested out [this variable](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/axes.js#L3839) but it doesn't seem to accurately account for the space between the axis tick labels and the axis title. Is there anything that I'm missing here? 

I've multiplied `approxTitleDepth(ax)` by 2 here since it seems to work based on visual inspection of a number of cases, but it does feel like a bit of a workaround.